Simple-step-by-step-setup-instructions-for-non-Linux-users.md

```diff
@@ -0,0 +1,225 @@
+# **Simple step-by-step Supernode setup instructions for non-Linux users**
+---
+This is the formal version of the `Alpha3 Supernode Guide for Dummies` by yidakee (@el_duderino_007) dedicated to the Community Tree development branch of Graft Network.
+
+**_This is not an official guide to be used with the main development branch_**
+
+## General considerations
+
+* This is a bare minimum install recommended for linux newbies and troubleshooting purposes.
+
+* This setup is **not recommended for mainnet**, as it is very a insecure server setup.
+
+* This guide is not intended for linux educational purposes, rather a simple copy/paste procedure to get your Supernode up an running and see it working properly and learn Supernode management, and later be able to follow more advanced setup guides.
+
+## Requirements
+
+* Any modern machine with at least 2 GB per core.
+* VPS (_virtual private server_) recommended.
+* Unlimited bandwidth - current implementation consumes roughly 15GB/day.
+
+
+## Brief list of VPS poviders
+
+
+| URL           | Specs         | Bandwidth | Storage | Price/month  |
+| ------------- |:-------------:| :-------------:| :-----:| :-----:|
+| https://vultr.com | 2vCPU 4GB | 3 TB | 60GB | $20 |
+| https://try.digitalocean.com | 2vCPU 4GB| 4 TB | 80GB     |   $100 Free Trial |
+| https://www.hetzner.com/ | 2vCPU 4GB | 20TB  | 40TB    |    4,90€|
+| https://contabo.com/ | 4 CPU 8GB | Unlimited | 200 GB      |    4,99€ |
+| https://www.linode.com/ |2vCPU 4GB | 4TB | 80GB      |    $20 |
+| https://www.ovh.com/ | 2vCPU 7GB | ?  | 50GB    |    $26.40 |
+
+
+* Start by selecting a VPS provider, spin up a node selecting Ubuntu 18.04. The Community Dev Tree accepts other distributions, but for this guide we will focus on Ubuntu 18.04.
+
+* Once you’ve fired up your node, `ssh` into the server and perform typical maintenance (note, it may not look exactly the same on your machine)
+
+
+    sudo apt-get update
+
+![1](simple-setup-images/1.png)
+
+
+    sudo apt-get upgrade
+
+![2](simple-setup-images/2.png)
+
+* If this window shows up, select the first option
+
+![3](simple-setup-images/3.png)
+
+* We will now install git
+
+
+    sudo apt-get install -y git
+
+![4](simple-setup-images/4.png)
+
+* Clone Community Tree Graft Supernode testnet code from GitHub repository
+
+      git clone --recursive https://github.com/graft-community/graft-ng.git
```

Comment:
We can simplify this a little bit to avoid needing the following `checkout` and `submodule` steps:

```
git clone --recursive -b gc-alpha https://github.com/graft-community/graft-ng.git
```

(also note that it should be on the gc-alpha branch rather than the alpha3 branch).